### ISSUE TITLE
Issue with copy-to on sub-topics with locktitles

### ISSUE THREAD

**Issue by arronvarga** [dita-ot/dita-ot#3372]
## Expected Behavior
The topic should use the navtitle defined in the map when @locktitle=yes.

Note: @copy-to and @locktitle works when used at the same level as the original topic.

## Actual Behavior
Instead topic is using the title of the first instance of the topic when topic is referenced as a sub-topic.

## Possible Solution
Fix copy-to to use the navtitle defined in the map

## Steps to Reproduce
Create ditamap with topic re-used as a sub topic

<map class="- map/map "
    domains="(map bookmap) (map mapgroup-d) (map subjectScheme) (topic abbrev-d) (topic delay-d) a(props deliveryTarget) (map ditavalref-d) (map glossref-d) (topic hazard-d) (topic hi-d) (topic indexing-d) (topic markup-d) (topic pr-d) (topic relmgmt-d) (topic sw-d) (topic ui-d) (topic ut-d) (topic markup-d xml-d) (topic xnal-d) a(loc ixia_locid)"
    id="axh1430149607893" title="Arron Test Map" xml:lang="en-us">

    <topicref class="- map/topicref " ixia_locid="5" keys="axk1425415118966"
        href="axk1425415118966.xml" type="task">
        <topicmeta class="- map/topicmeta ">
            <navtitle class="- topic/navtitle " ixia_locid="7">Performing reading tasks</navtitle>
            <linktext class="- map/linktext " ixia_locid="8">Performing reading tasks</linktext>
            <shortdesc class="- map/shortdesc ">Whether dictating with online speech recognition,
                digital dictation, or typing reports, you report on studies by way of reading tasks.
                A single reading task can contain one or more studies for which you create a single
                report.</shortdesc>
        </topicmeta>
    </topicref>
    <topicref class="- map/topicref " copy-to="second-axk1425415118966.xml" ixia_locid="11"
        keys="second-axk1425415118966" locktitle="yes" href="axk1425415118966.xml" type="task">
        <topicmeta class="- map/topicmeta ">
            <navtitle class="- topic/navtitle " ixia_locid="12">Arron Testing Copy-to and
                Locktitle</navtitle>
            <linktext class="- map/linktext " ixia_locid="13">Arron Testing Copy-to and
                Locktitle</linktext>
            <shortdesc class="- map/shortdesc ">Whether dictating with online speech recognition,
                digital dictation, or typing reports, you report on studies by way of reading tasks.
                A single reading task can contain one or more studies for which you create a single
                report.</shortdesc>
        </topicmeta>
        <topicref class="- map/topicref " copy-to="third-axk1425415118966.xml" ixia_locid="14"
            keys="third-axk1425415118966" locktitle="yes" href="axk1425415118966.xml" type="task">
            <topicmeta class="- map/topicmeta ">
                <navtitle class="- topic/navtitle ">Arron 2 Testing Copy-to and
                    Locktitle</navtitle>
                <linktext class="- map/linktext " ixia_locid="16">Arron 2 Testing Copy-to and
                    Locktitle</linktext>
                <shortdesc class="- map/shortdesc ">Whether dictating with online speech
                    recognition, digital dictation, or typing reports, you report on studies by way
                    of reading tasks. A single reading task can contain one or more studies for
                    which you create a single report.</shortdesc>
            </topicmeta>
        </topicref>
    </topicref>
    <topicref class="- map/topicref " copy-to="fourth-axk1425415118966.xml" ixia_locid="14"
        keys="fourth-axk1425415118966" locktitle="yes" href="axk1425415118966.xml" type="task">
        <topicmeta class="- map/topicmeta ">
            <navtitle class="- topic/navtitle ">Arron 3 Testing Copy-to and
                Locktitle</navtitle>
            <linktext class="- map/linktext " ixia_locid="16">Arron 3 Testing Copy-to and
                Locktitle</linktext>
            <shortdesc class="- map/shortdesc ">Whether dictating with online speech
                recognition, digital dictation, or typing reports, you report on studies by way
                of reading tasks. A single reading task can contain one or more studies for
                which you create a single report.</shortdesc>
        </topicmeta>
    </topicref>
</map>

## Copy of the error message, log file or stack trace
None

## Environment
* DITA-OT version: 3.3.4
* Operating system and version:windows
* How did you run DITA-OT?
oxygen
* Transformation type:
 HTML5

**Comment by robander** [comment]
I think this is a general problem in the module that collects metadata and cascades metadata. We had some issues open with that in the past, where using the same file results in overlapping metadata, with a result where the two copies do not use the right version.

I reproduced this with the `copy-to` structure above; I also tried another branch with the same original topic (no `copy-to`) and the locked title is also ignored there.

[3372.zip](https://github.com/dita-ot/dita-ot/files/3819038/3372.zip)
